pkg/apis/core/types.go

```diff
@@ -3148,8 +3148,8 @@ type PodAntiAffinity struct {
 	// most preferred is the one with the greatest sum of weights, i.e.
 	// for each node that meets all of the scheduling requirements (resource
 	// request, requiredDuringScheduling anti-affinity expressions, etc.),
-	// compute a sum by iterating through the elements of this field and adding
-	// "weight" to the sum if the node has pods which matches the corresponding podAffinityTerm; the
+	// compute a sum by iterating through the elements of this field and subtracting
```

Comment:
I'll let scheduling reviewers +1 the change first, but then make the same adjustment to the godoc in the core/v1/types.go file and run `make update` to regenerate API docs, etc

Reply:
New comment is correct.

/approve on scheduling side.

Reply:
@liggitt do we have an example updating api and doc? Just want to compare what needs to be updated; `make update` fails in some steps for me and I'm not sure if I can just ignore them.

Reply:
You can compose one commit that contains the "manual" changes only, in particular:

- pkg/apis/core/types.go
- staging/src/k8s.io/api/core/v1/types.go

For failures, if you cannot figure out, I can generate for you.

Reply:
make sure you've fetched tags from github (`git fetch $upstream-remote-name --tags`)

Reply:
> E0319 16:01:46.656089  583388 run.go:72] "command failed" err="[emulation version 1.24 is not between [1.31, 1.24.0-alpha.1.23735+f3158785791afa], minCompatibilityVersion version 1.23 is not between [1.31, 1.24]]"

cc @siyuanfoundation @Jefftree on the emulation-version / openapi error message in case there's something beyond getting updated tags locally that needs fixing

Reply:
Anyway, I just realized that the test failure outputs the diff against expected doc. I simply `patch`ed the diff on my commit then pushed.

Reply:
> make sure you've fetched tags from github (`git fetch $upstream-remote-name --tags`)

Yep this resolved the issue for me (but makes no diff after I've manually applied the patch taken from test failure). Thanks.